api/pkg/common/common.go

```diff
@@ -176,6 +178,35 @@ func InitCtxWithAuthInfo(request *restful.Request) context.Context {
 	return ctx
 }
 
+// TODO: Can be optimized with above function InitCtxWithAuthInfo()
+func InitCtxWithAuthInfoNonAdmin(request *restful.Request) context.Context {
+	actx := request.Attribute(c.KContext).(*c.Context)
+	var ctx context.Context
+	if actx.IsAdmin {
```

Comment:
Why do we need to get the CTX for admin here? This function is only for non-admin, right?

Reply:
List Tiers can have two user
1. admin
2. user
So both case required